packages/teams-js/src/public/monetization.ts

```diff
@@ -67,8 +67,7 @@ export namespace monetization {
     param2?: PlanInfo,
   ): Promise<void> {
     let callback: ((error: SdkError | null) => void) | undefined;
-    /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-    let planInfo: PlanInfo;
+    let planInfo: PlanInfo | undefined;
```

Comment:
>| undefined [](http://example.com/codeflow?start=27&length=11)

This variable was previously defined but not set so it already was always `undefined` to start.